examples/global_consumer/tsconfig_gen.json.expected

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"allowJs":false,"declaration":false,"declarationMap":false,"emitDeclarationOnly":false,"noEmit":false,"resolveJsonModule":null,"sourceMap":false},"files":["./index.ts", "./../../examples/global_dts_generated_tsconfig/index.d.ts", "./../../examples/global_dts_generated_tsconfig/global.d.ts"]}
```

Comment:
While these .expected files are not pretty I think it is important to watch out for these relative paths changing without us noticing. The `./` for files in this package and the `../../` for files outside the package.

I want to prevent us not noticing the `if .startswith(local` breaking and these all becoming the `../../` paths.